datasets/allen-cell-imaging-collections.yaml

```diff
@@ -72,31 +73,47 @@ DataAtWork:
+      AuthorName: Kaytlyn A. Gerbin, Tanya Grancharova, Rory M. Donovan-Maiye, Melissa C. Hendershott, Helen G. Anderson, Jackson M. Brown, Jianxu Chen, Stephanie Q. Dinh, Jamie L. Gehring, Gregory R. Johnson, HyeonWoo Lee, Aditya Nath, Angelique M. Nelson, M. Filip Sluzewski, Matheus P. Viana, Calysta Yan, Rebecca J. Zaunbrecher, Kimberly R. Cordes Metzler, Nathalie Gaudreault, Theo A. Knijnenburg, Susanne M. Rafelski, Julie A. Theriot, and Ruwanthi N. Gunawardane
       AuthorURL: https://www.alleninstitute.org/what-we-do/cell-science/about/team/
     - Title: A comprehensive analysis of gene expression changes in a high replicate and open-source dataset of differentiating hiPSC-derived cardiomyocytes
       URL: https://doi.org/10.1038/s41598-021-94732-1
-      AuthorName: Tanya Grancharova*, Kaytlyn A. Gerbin*, Alexander B. Rosenberg, Charles M. Roco, Joy E.Arakaki , Colette M. DeLizo, Stephanie Q. Dinh, Rory M. Donovan‑Maiye, Matthew Hirano, Angelique M. Nelson, Joyce Tang, Julie A. Theriot, Calysta Yan, Vilas Menon, Sean P. Palecek, Georg Seelig & Ruwanthi N. Gunawardane
+      AuthorName: Tanya Grancharova, Kaytlyn A. Gerbin, Alexander B. Rosenberg, Charles M. Roco, Joy E. Arakaki , Colette M. DeLizo, Stephanie Q. Dinh, Rory M. Donovan-Maiye, Matthew Hirano, Angelique M. Nelson, Joyce Tang, Julie A. Theriot, Calysta Yan, Vilas Menon, Sean P. Palecek, Georg Seelig & Ruwanthi N. Gunawardane
       AuthorURL: https://www.alleninstitute.org/what-we-do/cell-science/about/team/
     - Title: A deep generative model of 3D single-cell organization
       URL: https://doi.org/10.1371/journal.pcbi.1009155
-      AuthorName: Rory M. Donovan-Maiye, Jackson M. Brown, Caleb K. Chan, Liya Ding, Calysta Yan, Nathalie Gaudreault, Julie A. Theriot, Mary M. Maleckar, Theo A. Knijnenburg & Gregory R. Johnson
+      AuthorName: Rory M. Donovan-Maiye, Jackson M. Brown, Caleb K. Chan, Liya Ding, Calysta Yan, Nathalie Gaudreault, Julie A. Theriot, Mary M. Maleckar, Theo A. Knijnenburg, Gregory R. Johnson
       AuthorURL: https://www.alleninstitute.org/what-we-do/cell-science/about/team/
     - Title: Systematic gene tagging using CRISPR/Cas9 in human stem cells to illuminate cell organization
       URL: https://doi.org/10.1091/mbc.e17-03-0209
-      AuthorName: Brock Roberts*, Amanda Haupt*, Andrew Tucker, Tanya Grancharova, Joy Arakaki, Margaret A. Fuqua, Angelique Nelson, Caroline Hookway, Susan A. Ludmann, Irina A. Mueller, Ruian Yang, Rick Horwitz, Susanne M. Rafelski & Ruwanthi N. Gunawardane
+      AuthorName: Brock Roberts*, Amanda Haupt*, Andrew Tucker, Tanya Grancharova, Joy Arakaki, Margaret A. Fuqua, Angelique Nelson, Caroline Hookway, Susan A. Ludmann, Irina A. Mueller, Ruian Yang, Rick Horwitz, Susanne M. Rafelski, and Ruwanthi N. Gunawardane
```

Comment:
I think we decided to go with ampersands not the word and. Please don't change this entry.